lib/runner/variant-reassignment.js

```diff
@@ -57,10 +59,10 @@ export default class VariantReassignment {
         await this._processProductDraft(productDraft, products)
       } catch (e) {
         this.logger.error(
-          'Error while processing productDraft %j, retrying.',
+          'Error while processing productDraft %j',
           productDraft.name, e.stack
```

Comment:
why just stack and not whole error object? Also if callback is not defined you will log same error twice (here and withing _getDefaultErrorCallback)?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/61/commits/1a00fa093185906a3f690d932afd5d880b039495